protocol/scripts/deploys/governance/contract-addresses.ts

```diff
@@ -0,0 +1,38 @@
+import { network } from "hardhat";
+
+export interface DeployedContracts {
+    MULTISIG: string;
+    TEMPLAR_NFT: string;
+    ELDER_ELECTION: string;
+}
+
+export const DEPLOYED_CONTRACTS: { [key: string]: DeployedContracts } = {
```

Comment:
It might be nice to have these in the same file as the other addresses (deploys/helpers.ts), everything in one place

Reply:
@frontier159 thoughts? I'm new to the structure of the deploy scripts etc.

Reply:
The reason I split it out is because there would be a tonne of empty strings within `helpers.ts` because we're going multichain. I walked through with butler and agreed that domain specific makes more sense in this case.

@butlerji had thoughts to bundle his PR on updating deployment in here too. This may/may not happen in time though...